exporter/loadbalancingexporter/metadata.yaml

```
@@ -25,8 +25,40 @@ tests:
           - backend-3:4317
           - backend-4:4317
   expect_consumer_error: true
-  goleak:
-    ignore:
-      top:
-        # See https://github.com/census-instrumentation/opencensus-go/issues/1191 for more information.
-        - "go.opencensus.io/stats/view.(*worker).start"
+
+telemetry:
+  metrics:
+    loadbalancer_num_resolutions:
+      enabled: true
+      description: Number of times the resolver triggered a new resolutions.
+      unit: "{resolutions}"
+      sum:
+        value_type: int
+        monotonic: true
+    loadbalancer_num_backends:
+      enabled: true
+      description: Current number of backends in use.
+      unit: "{backends}"
+      gauge:
+        value_type: int
+    loadbalancer_backend_latency:
+      enabled: true
+      description: Response latency in ms for the backends.
```

```suggestion
      description: Response latency of the backends.
```
Nit: Not sure if we need to include the unit here. Feel free to drop though if it's more clear to include it in addition to the unit. 👍

---

Personally, when looking at a Prometheus metric at /metrics I always appreciate to have the unit described there 🙂